x/staking/proto/cosmos/staking/v1beta1/query.proto

```diff
@@ -139,10 +139,20 @@ message QueryValidatorsRequest {
   cosmos.base.query.v1beta1.PageRequest pagination = 2;
 }
 
+message ValidatorAdditionalInfo {
+  // consensus_address is the consensus address of the validator.
+  string consensus_address = 1;
+}
+
+message ValidatorInfo {
+  Validator               validator       = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+  ValidatorAdditionalInfo additional_info = 2;
+}
```

Comment:
nit: missing "since sdk version".

Reply:
Also, maybe we don't need the ValidatorAdditionalInfo message, we can just add the fields to ValidatorInfo

Reply:
@facundomedica when you say "since SDK version", are you referring to which version it has been available since?

Reply:
Yes like this:

```
  string query = 6 [(cosmos_proto.field_added_in) = "cosmos-sdk 0.50"];

```

and at the message level:

```
  option (cosmos_proto.message_added_in) = "cosmos-sdk 0.46";
```

Reply:
@facundomedica I did it, but how do you know what the next version will be to put it in the code? I'm not sure about the version that I put

Reply:
You should use 52